custom_components/frigate/camera.py

```diff
@@ -219,7 +211,6 @@ def __init__(
                     f"rtsp://{URL(self._url).host}:8554/{self._cam_name}"
                 )
         elif self._camera_config.get("rtmp", {}).get("enabled"):
```

Comment:
rtmp is no longer possible, right? Maybe this part  can be removed too?

Reply:
But it can also be done in a separate PR.

Reply:
[As discussed, will do a followon PR]

Reply:
Depends what we want the min supported Frigate version to be, but probably safe to say everyone should be off 0.11 by now